src/ontology/edits/oeo-physical.omn

```diff
@@ -763,6 +955,17 @@ Genreally, flows in the model can be controlled/optimized."@en,
         Line
     
     
+Class: DammedWater
+
+    Annotations: 
```

Comment:
The subclasses of water need some attention. The def of dammed water is on dams, not on dammed water!!! 
But this is off-topic here.

Reply:
Maybe open a different issue? :-)

Reply:
I think we should leave water and its subclasses in this PR as it is and shift it to another issue. Either #43 or #78 or a completely new issue.